test/auro-combobox.test.js

```diff
@@ -153,7 +150,7 @@ describe('auro-combobox', () => {
   it('navigates menu with up and down arrow keys', async () => {
     const el = await defaultFixture();
 
-    const dropdown = el.shadowRoot.querySelector('auro-dropdown');
+    const dropdown = el.shadowRoot.querySelector('combobox-dropdown');
```

Comment:
we need to preserve `auro` in the component names.

Let's rename all these to `auro-comboboxdropdown`

Reply:
Please see comment above.